tests/base.py

```diff
@@ -27,8 +27,10 @@ def find_unused_port():
 
 
 class BaseTest(unittest.TestCase):
+
```

Comment:
We use no blank line after `class` and `def` for now, following `asyncio` code style.
If it's ok to Guido then it's perfect for us also.